doc/classes/GDExtension.xml

```diff
@@ -1,49 +1,64 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="GDExtension" inherits="Resource" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		A shared native library for GDExtension.
```

Comment:
Making these brief descriptions are is so hard :-)

While this is accurate, I feel like using fewer adjectives makes this a little easier to understand, especially for less technical folks:

```suggestion
		A native library for GDExtension.
```